contracts/CO30_Jonty_JackalsLoot.Woodland_ACR/edit_me/briefing/mission.html

```diff
@@ -0,0 +1 @@
+A CDF convoy transporting vital military assets was ambushed in Bystrica. Intelligence suggests Miyović's forces are responsible and have stashed the stolen assets in an undisclosed location within the city. Synixe contractors are tasked with linking up with Colonel Artyom's QRF at the ambush location. Colonel Artyom will provide them with critical intel on the capabilities of the assets, and potentially pinpoint leads on where Miyović's forces might have stashed them. The CDF has a dedicated support convoy on standby to assist you in securing the assets. Once you've located the assets, establish contact with the CDF command. Upon confirmation of the location, the support convoy will mobilize and provide additional firepower and manpower to secure the site.
```

Comment:
```suggestion
A CDF convoy transporting vital military assets was ambushed in Bystrica. Intelligence suggests Miyović's forces are responsible and have stashed the stolen assets in an undisclosed location within the city. Synixe Contractors is tasked with linking up with Colonel Artyom's QRF at the ambush location. Colonel Artyom will provide us with critical intel on the capabilities of the assets, and potentially pinpoint leads on where Miyović's forces might have stashed them. The CDF has a dedicated support convoy on standby to assist you in securing the assets. Once we've located the assets, establish contact with the CDF command. Upon confirmation of the location, the support convoy will mobilize and provide additional firepower and manpower to secure the site.
```
The briefings are written by us for us, so we should use first person language